packages/core/src/DropDownMenu/DropDownMenu.tsx

```diff
@@ -80,6 +85,7 @@ export const HvDropDownMenu = (props: HvDropDownMenuProps) => {
     expanded,
     defaultExpanded = false,
     category = "secondaryGhost",
+    variant = "secondaryGhost",
```

Comment:
If the user doesn't use the new `variant` and it's still using `category`, the default value for the `variant` property will always override the value set for `category`. I believe not setting a default value for `variant` should fix this for now.